flutter_local_notifications/lib/src/platform_specifics/android/method_channel_mappers.dart

```diff
@@ -234,7 +234,8 @@ extension AndroidNotificationDetailsMapper on AndroidNotificationDetails {
         'visibility': visibility?.index,
         'timeoutAfter': timeoutAfter,
         'category': category,
-        'additionalFlags': additionalFlags
+        'fullScreen': fullScreenIntent,
```

Comment:
missed renaming the string to `fullScreenIntent` so both the Dart and Android side match